containers/zammad/docker-entrypoint.sh

```diff
@@ -19,7 +19,9 @@ set -e
 : "${ZAMMAD_RAILSSERVER_PORT:=3000}"
 : "${ZAMMAD_WEBSOCKET_HOST:=zammad-websocket}"
 : "${ZAMMAD_WEBSOCKET_PORT:=6042}"
+: "${ZAMMAD_INSTALL_RSYNC_PARAMS:=--no-perms --no-owner}"
```

Comment:
i don't see this used anywhere?

Reply:
Please ignore that! All before this commit (https://github.com/zammad/zammad-docker-compose/pull/150/commits/f839feccb4eaef06c90778757ae5c4d5913b5e80), was testings to solve broken CI pipeline.

Reply:
The commit quoted above resolve the first request (export rsync params to a generic var). So all other PRs can be rejected if you prefer.